### PR TITLE
Updated the app version

### PR DESCRIPTION
**Pull request by hardikhdholariya**
Updated the app version to 4.3.0

### CODE REVIEW

**Review by mahirchavda — APPROVED** [review]

File: cyences_app_for_splunk/default/app.conf [hunk 1]
```diff
@@ -12,7 +12,7 @@ label = Cyences App for Splunk
 [launcher]
 author = CrossRealms International
 description = Cyences App for Splunk built by CrossRealms International
-version = 4.2.1
+version = 4.3.0
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/app.conf:15]: Just grep to see that's the place we have the version number mentioned, just to ensure.

Comment (VatsalJagani) [inline cyences_app_for_splunk/default/app.conf:15]: Change version number in JS project for configuration page as well.

Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/app.conf:15]: Didn't get it? do you mean on cyences settings page anywhere? greped it but do not found.

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/app.conf:15]: https://github.com/CrossRealms/Splunk-Cyences-App-for-Splunk/blob/master/product_setup_page/package.json

Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/app.conf:15]: Discussed with mahir, it is not required as we are not using it anywhere.

**Review by VatsalJagani — APPROVED** [review]

**Pull request merged**